skin_meta_api.lua

```diff
@@ -56,7 +56,7 @@ function skin_class:set_hand(hand)
 end
 
 local ALPHA_CLIP = minetest.features.use_texture_alpha_string_modes and "clip" or true
-function skin_class:set_hand_from_texture()
+function skin_class:set_hand_from_texture(is_new)
```

Comment:
This is not future-proof. Instead, pass a version string like `skin_format` below and handle all supported kinds in this function. The possible values should also be documented in API.md.